README.md

```diff
@@ -440,16 +430,78 @@ Those SNAPSHOTS were built and deployed last night by Jenkins jobs.
 
         Note that using `-nsu` will also make the build faster.
 
+### **Set of projects:**
+
+If you want, instead, to build a set of projects that are dependent on each other, we suggest using the [build-chain action](https://github.com/kiegroup/github-action-build-chain) tool. This tool allows you to build multiple projects from different repositories in one single command.
+
+* Install **build-chain-action** npm tool (you must have node/npm installed):
+    
+    ```shell
+    $ npm i @kie/build-chain-action
+    ```
+
+Now in according to what you need you only have to run the *build-chain-action* tool with proper arguments. Here we will provide the most common use cases you might face into. Check [build-chain](https://github.com/kiegroup/github-action-build-chain#github-action-build-chain) documentation if you need further details on commands and arguments.
```

Comment:
I would suggest to keep this simple and specify just the branch one. I suggest to move pull request part to https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/.github/pull_request_template.md
wdyt?

If you agree please reformulate this part and commit new changes to pull_request_template.md file. 

Additionally I would just specify build chain command to build main branch with a note saying (something like)

> consider to change `main` by the branch/tag to build

and I would refer to https://github.com/kiegroup/github-action-build-chain#execution-build-action---branch-flow-arguments for more details.

Reply:
I completely agree on moving the `pull request` in https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/.github/pull_request_template.md, it makes sense having only the branch flow here and move the pull request related things there.